packages/agoric-cli/src/lib/wallet.js

```diff
@@ -272,44 +271,3 @@ export const makeWalletUtils = async (
     pollOffer,
   };
 };
-
-/**
- * @param {{
- *   brand: Record<string, Brand>,
- *   vbankAsset: Record<string, { brand: Brand, displayInfo: DisplayInfo }>,
- * }} agoricNames
- * @param {(msg: string) => Error} makeError error constructor
- * @returns {(a: string) => Amount<'nat'>}
- */
-export const makeParseAmount =
```

Comment:
removing this export is a breaking change to the `@agoric` package, right?

Reply:
No, the Agoric CLI is a UI tool and no exports are supported.

Reply:
A couple things could make that more clear:

 - something in the README.md
 - in `package.json`, something like `"exports": []` 

The the README has a bunch of stuff about running an IBC relayer that seems out of place in contexts such as https://www.npmjs.com/package/agoric